src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1313,6 +1312,11 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		this._webglAddon = new Addon();
 		try {
 			this._xterm.loadAddon(this._webglAddon);
+			this._webglAddon.onContextLoss(() => {
+				this._storageService.store(SUGGESTED_RENDERER_TYPE, 'dom', StorageScope.GLOBAL, StorageTarget.MACHINE);
```

Comment:
Oh I meant we want to make sure this does _not_ happen, otherwise all future terminals will use dom until the user touches the setting.